client/src/components/menubar/clip/index.tsx

```diff
@@ -66,12 +61,14 @@ const Clip = React.memo((props) => {
               style={{
                 cursor: "pointer",
               }}
+              ref={tooltipRef}
+              {...tooltipProps}
             />
           </Tooltip>
-        }
+        )}
         position={Position.BOTTOM_RIGHT}
-        onOpening={handleClipOpening}
-        onClosing={handleClipClosing}
+        onOpening={() => handleClipOpening}
```

Comment:
I assume `handleClipOpening` is already a function, why do we need to wrap it in an arrow function 💡 ?